daphne/dapf_test.sh

```diff
@@ -0,0 +1,97 @@
+#/bin/bash
+#
+# Copyright (c) 2022 Cloudflare, Inc. All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+
+# Script for demonstrating how `dapf` is used. To run it, you need to have
+# `dapf` in your $PATH, e.g., by running `cargo install --path .`.
```

Comment:
I think this would be premature, I don't want to have to make promises about `dapf` usage at this stage. I would like to experiment with it a little bit and see if there are any rough edges.